docs/features/files.md

```diff
@@ -18,3 +18,19 @@ new GenericContainer(...)
                                       "/etc/redis.conf",
                                       BindMode.READ_ONLY)
 ```
+
+## Copying files to and from containers
+
+Files can be copied into the container before startup, or can be copied from the container after the container has started.
+
+### Copying to a container before startup
+
+<!--codeinclude-->
+[Copying files to a container](../../core/src/test/java/org/testcontainers/junit/CopyFileToContainerTest.java) inside_block:copyToContainer
+<!--/codeinclude-->
+
+### Copying a file from a running container
+
+<!--codeinclude-->
+[Copying files from a container](../../core/src/test/java/org/testcontainers/junit/CopyFileToContainerTest.java) inside_block:copyFromContainer
```

Comment:
For some reason the whole file got included:
https://deploy-preview-4661--testcontainers.netlify.app/features/files/

Reply:
I made it a bit better but did not fix it. Am I misunderstanding how our markdown codeinclude plugin works, or is there some issue with it? Can you have a look at the current effect in this PR @rnorth?

Reply:
The block name doesn't match:
https://github.com/testcontainers/testcontainers-java/pull/4661/files#diff-54ea1df08fc5691456ef50f20cdaddb8fca38841e7877ebc8ef5d3521cede928R49

`copyFromContainer` vs `copyFileFromContainer`

Reply:
This is outdated